app/src/main/java/com/kickstarter/ui/adapters/DiscoveryPagerAdapter.java

```diff
@@ -40,8 +41,12 @@ public void setPrimaryItem(final @NonNull ViewGroup container, final int positio
   }
 
   @Override
-  public @NonNull Fragment getItem(final int position) {
-    return DiscoveryFragment.newInstance(position);
+  public
+  @NonNull
+  Fragment getItem(final int position) {
+    DiscoveryFragment fragment = DiscoveryFragment.newInstance(position);
```

Comment:
looks like check style wants this to be `final`. we like to `final` all the things